group_vars/all/support.yml

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
I emailed @BjoernT to have his opinion.
It's indeed unnecessary to ship this file. 

I'd love see support interact on rpc-o, bring their variables and playbooks here.